ui/components/data-sources/AddDataSourceDialog.js

```diff
@@ -1,203 +1,62 @@
-import React, { Component } from "react";
-import {
-    Alert,
-    Modal,
-    Icon,
-    Button,
-    Form,
-    FormControl,
-    FormGroup,
-    Col,
-    InputGroup,
-    DropdownButton,
-    MenuItem
-} from "patternfly-react";
-import some from "lodash.some";
-
 import { graphql } from "react-apollo";
 
-import { InMemoryForms } from "./forms";
+import BaseDataSourceDialog from "./BaseDataSourceDialog";
 import { DataSourceType } from "../../graphql/types/DataSourceType";
 import CreateDataSource from "../../graphql/CreateDataSource.graphql";
 import GetDataSources from "../../graphql/GetDataSources.graphql";
 
 const INITIAL_STATE = {
     name: "",
     type: DataSourceType.InMemory,
-    err: "",
-    inMemoryValues: {
+    options: {
         timestampData: true
     },
+    err: "",
     validations: {
         name: null,
         type: "success",
-        inMemoryValues: "success"
```

Comment:
@darahayes I think I understand what you really meant in last PR with this field. At that time I didn't realize these "values" really belonged to `config.options`

Reply:
Sorry wrong line, I meant [line 28](https://github.com/aerogear/data-sync-ui/pull/29/files#diff-017239a63889a936cf36a053872fe929L28)